instagram/src/components/Post/Post.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from "prop-types";
+import '../Post/Post.css';
+
+class Post extends React.Component {
+  render() {
+    return (
+      <div className={"postContainer"}>
+        <div className={"postTop"}>
+          <img src={this.props.post.thumbnailUrl} alt="user thumbnail" />
+          <div className={'topUsername'}>{this.props.post.username}</div>
```

Comment:
Unless we need JSX to evaluate some JavaScript for us, we don't need curly braces for our classNames. Also be careful about mixing double and single quotes, it's best to pick one and stick with it.